changelogs/fragments/446-ms_access_policy.yml

```diff
@@ -0,0 +1,2 @@
+breaking_changes:
```

Comment:
In case a user does access_policy_type Hybrid Authentication then we are going to get error of "increaseAccessSpeed attribute is needed" this will fix that break. There is no other case when this was required and as per docs also this was the primary case.